storybook/design-system/src/pages/get-started/changelog/markdown/content.md

```diff
@@ -1,6 +1,22 @@
 ### Changelog
 All notable changes to this project will be documented in this file.
 
+#### [v1.3.23](https://github.com/AxaGuilDEv/react-toolkit/compare/v1.3.22...v1.3.23)
```

Comment:
Changelog is automatically generated, maybe we should modify the template to add the diff automatically